cmd/client_changefeed.go

```diff
@@ -739,3 +742,19 @@ func getUpstreamCredential() *security.Credential {
 		KeyPath:  upstreamSslKeyPath,
 	}
 }
+
+func confirmChangefeedRemove(cmd *cobra.Command) error {
+	if noConfirm {
+		return nil
+	}
+	cmd.Printf("Confirm that you want to remove this changefeed [Y/N]\n")
+	var yOrN string
+	_, err := fmt.Scan(&yOrN)
+	if err != nil {
+		return err
+	}
+	if strings.ToLower(strings.TrimSpace(yOrN)) != "y" {
+		return errors.NewNoStackError("Abort remove changefeed!")
```

Comment:
if i type `N` to cancel the operation,  seems should not return error?

Reply:
I did this by referring to the [unsafe operation](https://github.com/pingcap/ticdc/blob/dfc046fd5dbe973a49eba4d31c376ed7634c3f82/cmd/client_unsafe.go#L116). But I think you are right, I will fix it.